### DISCUSSION TITLE
能够在视频播放页面，增加“自动展开评论回复”的功能吗？

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#4286] [功能建议 / Ideas]
现在好像也挺多的评论下的回复，是被收起来的。
希望能够直接展开回复，而不是需要逐个点击“共有n条回复，点击查看”

**Comment by Geeyun-JY3** [comment]
这真的不会卡死吗……

**Reply by kethoven** [reply]
我看他有些一条回复，也不展开显示。
感觉以前回复多的，也基本会展开几条回复，再提示查看更多。
现在好像基本不展开显示了。‘
应该是改了算法。很不习惯，’

**Reply by Geeyun-JY3** [reply]
阿瓦隆是这样的。
